debian/rules

```diff
@@ -84,3 +84,7 @@ override_dh_auto_install:
 	# Install NSS
 	# In Rust, HOST actually refers to the build target (see README.Debian in rustc)
 	install -Dm644 target/$(DEB_HOST_RUST_TYPE)/release/libnss_authd.so debian/authd/usr/lib/$(DEB_TARGET_GNU_TYPE)/libnss_authd.so.2
+
+	# Install systemd units
+	install -Dm644 systemd/authd.service debian/authd/lib/systemd/system/authd.service
```

Comment:
The day we add another unit, we will probably miss this. I would rather use a wildcard to install everything under the systemd/ directory.